---
layout: page
title: 

subtitle:
---

<style>
.pheader {
  font-size: 150%;
  color: #bf5700;
  font-weight: bold;
}

.subpheader {
  font-size: 120%;
  color: #005f86;
  font-weight: normal;
}
</style>



<center>
	<font size="+4"><b>RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP)</b></font><br>
	<font size="+2">Detailed online protocol</font><br>
Version 1, January 20, 2022
</center>

<font size="+1"><b>Table of Contents:</b></font>


<ol type="1">

	<li><a href="#overview">Overview</a></li>
	<li><a href="#materialslist">Materials list</a></li>
	<li><a href="#preexperimentalsetup">Pre-experimental setup</a></li>
	<li><a href="#riboitpprotocol">Ribo-ITP Protocol</a></li>
	<li><a href="#generalconsiderations">General Considerations</a></li>
	<li><a href="#otherconsiderations">Other Considerations</a></li>

</ol>

<!-- ###################################################################  -->

<br>
<br>
<center>
	<!-- <img src="/img/Figure1-web.jpg"> -->
	<img src="/img/protocols/riboitp/ribo_itp_schematic.jpeg">
</center>
<br>
<br>


<!-- ###################################################################  -->


<ol type="1">
  <li class="pheader" id="overview">Overview</li>

  
  <ol type="A">
	  
    <li class="subpheader" >Purpose</li>
    The purpose of this protocol is to detail the RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP) protocol described in Tonn et al. 2021 to maximize execution and reproducibility in other laboratories. Here, we detail all reagents and materials needed, steps to take for experimental setup, and the Ribo-ITP protocol, and general considerations. 
  </ol>
  
  <!-- OVERVIEW ENDS  -->





  <li class="pheader" id="materialslist">Materials list</li>

  <ol type="A">

    <li class="subpheader" >Materials</li>

    <i>??could insert spreadsheet with all reagents, including a tab for chemicals, oligonucleotides, etc. For transparency, could include manufacturer/part number and approximate cost for each item ?? </i>

    <li class="subpheader" >Purchasing molds for microfluidic chips</li>
    <i>??insert detailed information on how to order the molds from ProtoLabs??</i>
  </ol>

  <!-- MATERIALS LIST ENDS  -->





  <li class="pheader" id="preexperimentalsetup">Pre-experimental setup</li>
  
  <ol type="A">

    <li class="subpheader" >Buffer preparation</li>
    ??<i> insert a table of the buffer recipes outlined on the in-house protocol</i>  ??
  
    <li class="subpheader" >Gel purification of fluorescent DNA oligonucleotides</li>
    <i>  ?? Write protocol here for oligo purification  ??   </i>

    <li class="subpheader" >Mold setup for Ribo-ITP chips</li>
    <i> ?? Write protocol here for how to do the initial setup of the molds  ??  </i>


    <li class="subpheader" >Manufacturing Ribo-ITP chips</li>
    <ol type="1">
        <li> Mix Sylgard 184 PDMS monomer and curing agent (Ellsworth) at 10:1 (w/w) ratio.  </li>    
        <ol type="a">
	    <li> For initial setup of the molds, the PDMS mixture should be of sufficient volume to cover the entire mold (i.e. to cover the glass slide and the attached molds) and reach a depth of at least ~2 cm. </li>
	    <li> For subsequent experiments after the initial setup, prepare enough of the PDMS mixture to provide ~7 g of PDMS mixture to each two-lane mold.  </li>
        </ol>
        
	<br>
	<img src="/img/protocols/riboitp/01_supplies_for_mixing_PDMS.jpg">
	<br><br><br>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/ilK37HF0Lv0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
        <br>
	<br>

	<li> Degas the PDMS mixture in a desiccator connected to a vacuum chamber for ~30 min or until no visible air bubbles are seen.  </li>
	
	<br>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/4f1GB6H12do" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
        <br>
        <img src="/img/protocols/riboitp/05_merge_04_text.jpeg">        
        <br>
	<br>

        <li> Pour the degassed PDMS mixture evenly onto the molds. </li>
        <ol type="a">
	   <li> For initial setup of the molds, pour the PDMS mixture over the entire surface of the mold (i.e. to cover the glass slide and the attached molds) and reach a depth of at least ~ 2 cm. </li>
	   <li> For subsequent experiments after the initial setup, pour the PDMS mixture evenly over the exposed surfaces of the chips, ensuring that each gets approximately 7 g of PDMS mixture and reaches a depth of at least ~2 cm. </li>
	</ol>
	<br>    
	<iframe width="560" height="315" src="https://www.youtube.com/embed/bEBcXd6YLTM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
        <br>
	<br>

        <li> Degas the PDMS mixture in a desiccator connected to a vacuum chamber for ~30 min or until no visible air bubbles are seen. </li>
	<br>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/m6bHhIn1-Qo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	<br>
	<br>

        <li> Incubate the PDMS molds at 50°C overnight. </li>
        <br>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/K610zewU0GI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	<br>
	<br>

        <li> After overnight incubation, remove the molds from the incubator and touch the top of the molds with gloved fingers to ensure that the PDMS has solidified and is no longer sticky. If any stickiness or liquid PDMS is noted, return the molds to the 50°C incubator until curing is complete.  </li>

        <li> Using a razor blade, cut the PDMS slabs from the molds along the border of the chips.</li>
        <ol type="a">
	    <li> The border is indicated by red below.</li>
	    <img src="/img/protocols/riboitp/itp_chip_borders.png">
            <br> 
        <li> Take care to avoid cutting the molds and PDMS slabs in the areas of the channels to maintain their integrity </li> 
	<li> 3 PDMS slabs (6 lanes total) fit on a 5” x 4” glass slide.  </li>
	</ol>

        <li> Place the PDMS slabs feature-side up and punch out the five reservoirs using a 5 mm diameter biopsy punch. </li>
        <img src="/img/protocols/riboitp/09_supplies_for_cutting_baked_channels_from_molds.jpg"> 
	<br>
	<br>
	<ol type="a">
           <li> The borders of the reservoirs are indicated by red circles below. </li>
	   <img src="/img/protocols/riboitp/itp_chip_red_circles.png">
	   <br>
           <li> Cover channels with tape for storage, or continue to plasma cleaning.  </li>    
        </ol>

        <li>Plasma clean the PDMS slabs and glass slide as follows:<br>
		<i>Important note: The plasma cleaning protocol may need to be optimized for different plasma cleaner setups. </i></li>

	<ol type="a">
	   <li> Turn on power of plasma cleaner and vacuum gauge. </li>	
	   <li> Place PDMS channel feature side up and glass slide (4’’ by 3’’; Ted Pella) in chamber (pre-clean the slide and channel with tape).  </li>
	   <li> Make sure the 3-way valve is closed (lever pointing down).  </li>
	   <li> While holding the door shut, turn on the vacuum pump.  </li>
	   <li> Let the vacuum pump down to 0.25 Torr (can take up to 15 min).  </li>
	   <iframe width="560" height="315" src="https://www.youtube.com/embed/FnjKj3wZuag" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	   <br>
	   <br>
	   <li> Open the 3-way valve to let air flow (lever pointing left). Let the pressure again reach 0.25 Torr.  </li>
	   <iframe width="560" height="315" src="https://www.youtube.com/embed/_RtIEDzwjVM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	   <br>
	   <br>
	   <li> Turn on the RF power to “HI” and treat the samples for 2 min. Bright violet plasma color should be observed.  </li>
	   <li> Turn off the RF and vacuum pump, and close the 2-way valve (lever pointing down).  </li>
	   <li> Slowly vent the chamber by opening the 3-way valve slightly (lever pointing slightly to the right).  </li>
	   <li> Should hear a hissing sound as the air evacuates. This should be done slowly (takes at least 1 min).  </li>
	   <li> Chamber is fully vented when pressure returns to atmospheric pressure  </li>
	   <iframe width="560" height="315" src="https://www.youtube.com/embed/fknTZiRrYzc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	   <br>
	   <br>
	   <li>Open the chamber and bring the plasma-treated surfaces together. You can gently push air bubbles from beneath the PDMS, but it is not necessary to push hard, as this can damage the channels. </li>
	   <iframe width="560" height="315" src="https://www.youtube.com/embed/tFFZ_alkxS4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	   <br>
	   <br>


	</ol>
	<li> Heat the chips at 80°C for a minimum of 15 minutes and up to overnight to enhance bonding.  </li>
	<img src="/img/protocols/riboitp/15_incubating_plasma_chip_80C_overnight.jpg">
	<br>
	<br>
	<li>Cover channels with tape for storage. </li>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/mcnpF5JZmlQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	<br>
	<br>
    </ol>




  </ol>


  <!-- PREEXPERIMENTAL LIST ENDS  -->




  <li class="pheader" id="riboitpprotocol">Ribo-ITP protocol</li>

  <ol type="A">

    <li class="subpheader" > Cell preparation: dilution method for cell culture </li>
    <i>*Use this method if performing Ribo-ITP with dilutions of K562 cells</i>
    
    <ol type="1">
        <li> Grow K562 cells in RPMI 1640 + 10% FBS (media).  </li>
	<li> Pellet cells at 300 g for 10 min. </li>
	<li> Resuspend cells with 5 mL media.  </li>
	<li> Wash cells once with 1x phosphate buffered saline (PBS) then resuspend in 1 ml PBS. </li>
	<li> Pellet cells in PBS at 300 g for 10 min. </li>
	<li> Remove PBS and resuspend pellet in Lysis Buffer Working Solution to get desired cell concentration in a final volume of 5 μL. We used a dilution equivalent to 100 cells in 5 μL of buffer.  </li>
	<li> To generate ribosome protected fragments, proceed to the section titled “Nuclease digestion for generation of ribosome protected fragments (RPFs) from diluted cell culture lysate”.  </li>
    </ol>

    <li class="subpheader" > Preparation of mouse oocytes and embryos.  </li>
    <i> *Use this method if performing Ribo-ITP with mouse oocytes or embryos. </i>

    <ol type="1">
        <li> Refer to Bill Shawlot’s methods described in Mouse oocyte isolation and the In vitro fertilization (IVF) using CAST/EiJ sperm sections in Single cell quantification of ribosome occupancy in early mouse development.  </li>

        <ol type="a">
	    <li> GV and MII collected from C57BL/6J mice (female).  </li>
	    <li> IVF performed in order to get 1, 2, 4 and 8-cell stage embryos (CAST/EiJ (male) x C57BL/6J).  </li>
	</ol>
	<li> Oocyte collection performed as done by <a href="https://link.springer.com/protocol/10.1007/978-1-4939-8603-3_7">Borensztein et al in Transcriptome Profiling of Single Mouse Oocytes (2018)</a>  </li>
	<ol type="a">
	    <li> This collection method minimizes the final buffer volume in the starting sample (&#60;1 μL each).  </li>
	    <li> Upon collection, samples are added to low bind 0.2 ml PCR tubes (Axygen) and submerged in liquid nitrogen. Samples are immediately transferred to -80°C for storage until sample processing, which takes place as soon as possible. We have found that the sooner the samples are processed after collection, the better our data. For the mouse samples in the ITP paper, samples were processed within 24 – 48 of collection. </li>
	    <li> To generate ribosome protected fragments, proceed to the section titled  “Nuclease digestion for generation of ribosome protected fragments (RPFs) from mouse oocytes, embryos” section.  </li>
	</ol>
    </ol>
    
    <li class="subpheader" > PDMS chip preparation for Ribo-ITP experiments  </li>
    <ol type="1">
        <li> Prepare prepolymer mixes (10% and 5% PPM) according to the recipes in the “buffers” section. Heating at 50°C and vortexing the prepolymer mix is necessary to fully dissolve the urea. Vortex again after addition of VA-086, and cover tubes with foil to reduce light exposure.  </li>
        <li> Attach PDMS chip to blue light source with tape.  </li>
        <li> Pretreat the channels of the ITP chip by filling the entire chip with each solution for the indicated amount of time. Empty and dry the channels between each step using a Pasteur pipette tip attached to a vacuum hose and liquid trap. </li>
        <ol type="a">
            <li> 1 min of RNaseZap (100% concentrate).  </li>
	    <li> Nuclease-free water rinse. </li>
	    <li> 1 min of 1 M NaOH. </li>
            <li> Nuclease-free water rinse.  </li>
            <li> 10 min of 10% (w/v) Benzophenone in Acetone.  </li>
	    <ol type="1">
	        <li> During benzophenone treatment, prepolymer mixes heated at 50°C for 10 min.  </li>
		<li> Ensure there are no bubbles present during the treatment; more of the solution can be pipetted into the channel during treatment to remove bubbles. </li>
            </ol>

            <li> 1 min of MeOH. </li>
            <li> 1 min of 0.1% Triton X-100 in nuclease-free water.  </li>
            <iframe width="560" height="315" src="https://www.youtube.com/embed/YkX4epOHu2Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> 
	    <br>
	    <br>
	</ol>

        <li> Polyacrylamide stacking </li>
	<ol type="a">
            <li> Secure chip to UV light source (turned off) with tape to avoid disturbing the prepolymer mix prior to polymerization. </li>
	    <li>Fill the 10% polyacrylamide channel with the 10% PPM by dial pipetting into the size-selection/10% PA channel through the elution well. </li>
            <iframe width="560" height="315" src="https://www.youtube.com/embed/Qfk3PsO9tng" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	    <br>
	    <ol type="1">
                <li>  Dial pipetting was performed by slowly turning the volume knob to the left to reduce the displayed volume in the micropipette.  </li>
                <li> Ensure that the prepolymer mixtures are being pipetted into the channel by aligning the pipette tip right at the channel entrance.  </li>
                <li> Take extra care to ensure the 10% PPM does not block branch channel 2 when loading.  </li>
            </ol>
	    <li>  Fill the extraction channel/5% polyacrylamide channel with the 5% PPM by dial pipetting through branch channel 2. Ensure the 5% PPM does not block branch channel 1 when loading as this will not prevent the loading of the lysate into the lysate channel.  </li>     
            <li> Expose chip containing the stacked PPM to UV light for one minute, followed by darkness for 30 seconds  </li>
            <li> Repeat UV exposure two more times for a total three-minute exposure.  </li>
	    <li> Fill all empty channels and reservoirs with LE storage buffer to avoid polyacrylamide dehydration. </li>
	    <iframe width="560" height="315" src="https://www.youtube.com/embed/NZiT7BpDdjQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	    <br>
	    <br>
		   
	</ol>

        <li> While waiting to perform ITP, cover the chips with a box or foil to protect from light. Chips are typically used within 30 min to 6 hours of preparation.  </li>
    </ol>
    
    <li class="subpheader" > Nuclease digestion for generation of ribosome protected fragments (RPFs) from diluted cell culture lysate. </li>
    * <i>Digestion should be timed so that the incubation step ends right before loading it into the lysate channel</i><br>
    **<i> When preparing ~6 samples in a day, process three at a time through digestion and ITP, then repeat digestion for the next three samples.  </i>

    <ol type="1">
       <li> Dilute MNase stock [2000 U/μL] (NEB). Dilute 1:50 in lysis buffer for 100 cell dilution experiments by adding 2 μl MNase stock to 98 μl lysis buffer.  </li>
       <li> Add 1 μl of the diluted MNase to the lysate, cap the PCR tube, and incubate in thermal cycler for 30 min at 37°C. </li> 
       <li> Dilute EGTA to 70 mM in lysis buffer and keep on ice.  </li> 
       <li> Add 1 μl of EGTA [70 mM] solution to the lysate to a final concentration of 10 mM.  </li>
       <ol type="a">
	       
          <li> If working with tubes, vortex samples and spin down.  </li> 
          <li> If working with plates, pipette to mix.  </li> 
       </ol>
       <li> Keep lysate on ice until Ribo-ITP. </li> 

    </ol>
    <li class="subpheader" > Nuclease digestion for generation of ribosome protected fragments (RPFs) from mouse oocytes, embryos.  </li>
    <ol type="1">
       <li> Add 6 μl of Lysis Buffer Working Solution directly to the frozen droplet and place it on a cold block on ice.   </li>
       <ol type="a">
          <li>  Dilute 2 μl of MNase in 98 μl Lysis Buffer to get the 1:50 dilution. A digestion mix was made by adding 4 μl of the 1:50 MNase to 20 μl Lysis Buffer. </li>
          <li> Repeat until all samples have Lysis Buffer. Pipette up and down ~5x to mix and then briefly centrifuge.  </li>
       </ol>
       <li> Samples are immediately digested at 37°C for 30 min.  </li>
       <li> After digestion, samples are immediately moved to ice and 1μL of EGTA [70 mM] is added for a final concentration of 10 mM.  </li>
       <li> Samples are vortexed and spun down, then kept on ice until ITP.  </li>
    </ol>
    
    <li class="subpheader" > Chip setup  </li>
    * <i> Keep pluronic solutions (LEp and TEp) and frozen pipette tips on ice to prevent premature gelling.  </i>
    <ol type="1">
       <li> Move chip to blue light source for ITP and secure with tape. </li>
       <li> Remove LE storage buffer from channels immediately prior to use using a Pasteur pipette attached to a vacuum hose and liquid trap.  </li>
       <li> Pipet 80 μL LEp through LE reservoir 3 with frozen tips so that it fills both LE reservoir 3 and the channel connecting it to the elution well. Frozen tips are required to prevent LEp from turning to gel early.  </li>

       <li> Add 30 μL LEp to LE reservoir 2 using frozen tips, ensuring contact with the polyacrylamide present in branch channel 2.  </li>
       <li> Pipet 20 μL running buffer into elution well.  </li>
       <li> Add 3 μL of fluorescent DNA markers to the digested lysate (7 μL). Prepare a 1:1 mastermix of gel-purified 19ntfl DNA and 35ntfl DNA [1 μM] in advance, aliquot, and store at -80°C.   </li>
       <li> Remove SDB from 50°C heatblock and add 11μL to the sample, then vortex and spin down.  </li>
       <li> Load  21 μL volume of the sample mixture was loaded into the lysate channel through branch channel 1 via dial-pipetting. </li>
       <iframe width="560" height="315" src="https://www.youtube.com/embed/Qfk3PsO9tng" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
       <br>
       <ol type="a">
          <li>  Take care to avoid bubbles Take care to avoid bubbles.  </li>
          <li>  A small amount of the lysate usually spills over into the TE reservoir. Similarly, once the lysate channel is full, it is ok to load a little excess volume directly into the TE reservoir, near the lysate channel.  </li>
       </ol>

       <li> Add 30 μL LEp to LE reservoir 1 using frozen tips, ensuring contact with the lysate present in branch channel 1.  </li>
       <li> Fill the TE reservoir with 70 μL TEp using frozen pipet tips. Care was taken to avoid bubbles near the lysate channel and to ensure good contact with lysate.  </li>
       <li> Place the negative electrode into the TE reservoir and secure it with tape.  </li>
       <li> Place the positive electrode into LE reservoir 3 and secure it with tape. Take care to maximize distance between electrodes and LEp or TEp, in order to prevent drying of the pluronic solutions.   </li>
       <iframe width="560" height="315" src="https://www.youtube.com/embed/4Ac6OSV3v38" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
       <br>
    </ol>
    
    <li class="subpheader" > RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP)  </li>
    <ol type="1">
        <li>Cover the BlueLight source (Clare Chemical Dark Reader Transilluminator) with its light filter then turn it on to enable fluorophore visualization. Turn off lights in order to visualize fluorophore well.  </li>
        <li> A Keithley 2410 Sourcemeter, controlled using MatLab was used to apply a constant 300 mA across the channel with a maximum voltage of 1.1 kV.  </li>
        <li> Apply current to the channel to start sample migration.  </li>
        <li> When the fluorescent band passes by Branch Channel 1, a secondary negative lead was added to LE Reservoir 1.  </li>
	<ol type="a">
	   <li>In a typical run, the starting voltage was ~250 V. The voltage initially decreases then increases as resistance in the channel becomes greater.
 </li>
	   <li>In an ideal run, we expect the marker DNAs to concentrate as a sharp band in the 5% PA gel. Upon entering the 10% PA gel, two bands (representative of each DNA marker) are expected to form.</li>
	   <li> Watch out for the timing of the DNA concentration. We expect the DNA to reach the 5% PA boundary by ~10 min. At the 13–14-minute mark, it should reach the 10% interface. Around 17 minutes, the first marker should be nearing the elution well. After cleaning well and exchanging the collection buffer for Diagenode Dephosphorylation Buffer (DB), sample elution is expected to take ~2 min. A typical run should take ~20-25 minutes. 
  </li>
	   <li> A good run will have minimal curvature of the DNA bands, and the two bands should separate more as they travel through the 10% PA gel.
  </li>
	</ol>
	<li> Pause current application when the lower fl-DNA marker (19ntfl) begins entering the elution well.  </li>
        <li> Discard the running buffer and wash the elution well twice with 30 μL nuclease-free water. Avoid leaving residual volume in the well.  </li>
        <li> Add 10 μL of 8:2 diluted Dephosphorylation Buffer (DB) (Diagenode) to the elution well. This is equivalent to 8 μL NFH2O + 2 μL DB, which is the starting buffer in the Diagenode D-Plex kit for library prep.  </li>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/u3mUuRIVTW0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
        <li> Resume the current until the upper fl-DNA marker (36ntfl) begins entering the elution well.  </li>
	<li> Pipet the 10 μL sample pipetted into a 0.2 ml low bind PCR tube and immediately freeze it at -80C until further processing.  </li>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/ldh-8tUr39E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	<iframe width="560" height="315" src="https://www.youtube.com/embed/R49TwVJf0eA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
    </ol>
    
    <li class="subpheader" > Library preparation for Ribo-ITP libraries  </li>
    <ol type="1">
        <li> Sequencing libraries were prepared using the Diagenode D-Plex Small RNA-seq kit (Diagenode) per manufacturer’s instructions, with the following minor modifications:  </li>
	<ol type="a">
	   <li> In the dephosphorylation rxn:  </li>
	   <ol type="1">
		   <li> used 1μL of a 1:1 mix of Dephosphorylation Reagent (DR) (Diagenode) and T4 PNK (NEB) instead of just 0.5 μL of DR.  </li>
		   <li> incubated the samples for 25 minutes at 37C instead of 15 min.  </li>
	   </ol>   
	   <li> For low input ITP samples, the RTPM was used as reverse transcription primer (not the RTPH). RTPH is used when we create “bulk” ribosome profiling libraries, with 25ng inputs as starting material.  </li>
	   <li> For low input ITP samples, a 1:2 dilution of the template switching oligo was required.   </li>
	   <li> After cDNA synthesis, ½ of the cDNA (13 μL) was stored as backup (-20C or long-term at -80C). The removed 13 μL was replaced with NFH2O and PCR proceeded as written. </li>
	   <li> 17 PCR cycles were used for low-input ITP samples (using ½ of the cDNA) in both the SI (single index) and UDI (unique dual index) modules of the kit.  </li>
	</ol>
        
	<li> 1 μL of a 1:4 dilution of the PCR product was assessed via High Sensitivity DNA bioanalyzer.   </li>    
        <ol type="a">
	    <li> A peak of ~200bp is expected when using the UDI module (~30nt inserts).  </li>
	    <li> A peak of ~190bp is expected when using the SI module (~30nt inserts).  </li>
	</ol>
	<li> The sample peak of interest was quantified by bioanalyzer.   </li>    
        <li> Samples are pooled such that an equimolar amount of the peak-of-interest was present in the pool. 40 μL of the least-concentrated sample is used. It is ok to skip pooling of the samples when dealing with low sample numbers. However, when many samples are being processed, it is helpful to pool “like samples” before bead purification and size selection. (e.g. make a pool of all GV samples, one pool of all MII samples, etc).   </li>    
        <li> Pools of PCR products were cleaned with Ampure XP beads (1.8x ratio) and eluted in 30 μL NFH2O.  </li> 
        <li> The entire elution was size selected on a 3% agarose cassette using the Blue Pippin system, targeting either 190 or 200 bp using “tight” settings.  </li>    
        <li> The 40 μL elution was collected from blue pippin and analyzed using High Sensitivity DNA bioanalyzer. Dilution is usually not necessary for this pool, but think about your expected concentration, which depends on your original sample pool. If needed, samples can be diluted prior to checking on bioanalyzer.  </li>    
    </ol>

  </ol>


  <!-- RIBO ITP PROTOCOL ENDS  -->






  <li class="pheader" id="generalconsiderations">General considerations</li>
 

  <ol type="A">
  
    <li class="subpheader" >  Working with RNA </li>
    <ol type="1">
        <li> Thoroughly clean your bench, pipets, etc with RNase zap before starting ITP.  </li>
        <li> Aliquot stocks of reagents in order to avoid risks of RNase contamination.  </li>
        <li> Change gloves often. </li>
        <li> Always use sterile, nuclease free, filtered pipet tips with low retention.  </li>
    </ol>
    
    <li class="subpheader" >  Preparing PDMS chips </li>
    <ol type="1">
        <li>
		 Some things to look out for if your PDMS chips aren’t working:
	</li>
	<ol type="a">
	    <li> If the liquid PDMS is underbaked after pouring it on the molds, it will be sticky and not bind to the glass channel properly after plasma cleaning. Ensure that the molds are not sticky after overnight incubation at 50C.   </li>
	    <li> If the edges of the channels are not properly binding to the glass after plasma cleaning, it may be time to prepare new molds. Older molds (~ 1 year old) start consistently having this problem, and replacing them with new ones immediately fixes the problem.
  </li>
	</ol>
    </ol>
    
    <li class="subpheader" >  Fluorescent markers </li>
    <ol type="1">
        <li> This protocol originally used fluorescently labeled RNA markers with 3’ ddC modifications. Despite the ddC blocking nucleotide, RNA markers consistently generated sequencing libraries (i.e negative controls just containing markers and no cellular input generated libraries; upon sequencing, these were clearly our markers). Over multiple tries, fresh RNA markers were re-ordered, single use aliquots were made, resuspension buffers were changed, etc… and they always degraded (even never-before-used aliquots!). IDT and Bio-Synthesis were unable to guarantee 100% pure RNAs. Additionally, these oligos are known to degrade in -80C over time. Due to these issues, the switch to ssDNA markers was made.  </li>
       <li>
       The switch to fluorescently labeled DNA markers with 3’ ddC modifications was made in Jan. 2020. The library prep kit (Diagenode D-Plex) was able to amplify ssDNAs with 3’ OH groups, but not the fl-DNAs with 3’ ddC (after they were gel purified in-lab). Again, since the synthesis companies aren’t able to guarantee 100% purity, it was required to gel purify the markers in house in order to gain higher purity (oligos with lengths n-1 or n-2 are the most common contaminants, which means the most common contaminants lack the 3’ ddC). When gel purifying, only the top ½ of the fluorescent band was extracted to remove ddC-lacking species in stock solutions.
       </li>
    </ol>   
    <li class="subpheader" >  Nucleases &amp; nuclease inhibition </li>
    <ol type="1">
        <li> Aliquot single-use vials of MNase to avoid freeze thaw cycles.  </li>
	<li> It would be great to enhance flexibility in the future by figuring out how to get ITP to work with different nucleases. Since MNase relies on Ca++ for function, EGTA (calcium quencher) served as an effective MNase inhibitor, and only minorly effected size selection/yield at Cf=10mM in ITP. Other nucleases, however, would need different inhibitors.   </li>
	<ol type="a">
            <li> We found that VRC migrated just ahead of our RPFs and was present in our collection fractions. VRC is known to inhibit downstream enzymatic reactions.   </li>
            <li> We tested proteinase K and found that it altered the digestion profile of our RNA samples. We think that it begins digesting the ribosome prior to fully inhibiting the nuclease, resulting in changes to the RNA profile (overdigestion).  </li>
	</ol>
    </ol>
    <li class="subpheader" >  Test experiments for beginners </li>
    <ol type="1">
       <li> Yield &amp; size selection  </li>
       <ol type="a">
           <li> One can run an RNA of choice (small RNA ladder or pre-digested/extracted RNA containing RPFs) through ITP and compare check yield on a gel, as compared to the input RNA. Can also assess size selection (exclusion of unwanted small/large fragments; yield of target fragments).  </li>
       </ol>
       <li> Ribosome profiling of ~100 K562 cells  </li>
       <ol type="a">
           <li> The first attempt with actual cells should be the 100 K562 cells (dilution method described in protocol).   </li>
           <li> Optimizations for new cell types/counts would likely be necessary.  </li>
       </ol>
    </ol>    
  </ol>

  <!-- GENERAL CONSIDERATIONS ENDS  -->









  <li class="pheader" id="otherconsiderations">Other considerations</li>
  <ol type="1">
     <li> pH is very important in ITP.  </li>
     <ol type="a">
         <li> Buffers have been optimized to have pH around 7.2. If the pH of buffers is incorrect, check the pH of the BisTris-HCl Master Mix and ensure it is at 7.2.  </li>
         <li> If size selection is poor or ITP seems to be running poorly, check the pH of your solutions; especially the lab-made ones. The 2.5M BisTris has a pH of &gt;= 10.  </li>
     </ol>
     <li> Aliquots are your best friend. They help avoid RNase contamination, freeze thaw cycling, etc.  </li>
     <li> Our protocol uses 1.5 μL of each 1μM fl-DNA marker. This can be hard to see, but we use this small amount in order to minimize risk of converting these to libraries. Marker amount can be scaled up depending on application.  </li>
  </ol>
  



</ol>


