---
layout: page
title: Single cell ribosome profiling for characterizing translation in development, cancer and immunology
html: True
image: /img/publications/Sankey.png
summary: We developed a novel microfluidic isotachophoresis method (Ribo-ITP) to measure ribosome occupancy from ultra-low input samples including single cells.
permalink: /lab_research-riboitp
---

<center>
  <!-- <h1> Single cell quantification of ribosome occupancy in early mouse development </h1> -->
</center>

<center>
<iframe width="560" height="315" src="https://www.youtube.com/embed/SSHRsWqrsdc?autoplay=1&mute=1&loop=1&playlist=SSHRsWqrsdc" frameborder="0" allow="autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</center>


<p>
We developed a new ribosome profiling approach that leverages the principles of microfluidics and isotachophoresis: <a href="https://www.nature.com/articles/s41586-023-06228-9">Ribo-ITP (Ozadam et al., Nature)</a>:
</p>

<p>
<i> 
Previous studies had been restricted to bulk measurements precluding precise determination of translation regulation including allele-specific analyses. To address this challenge, we developed a novel microfluidic isotachophoresis approach, named RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP), and characterized translation in single oocytes and embryos during early mouse development. Our high coverage measurements enabled the first analysis of allele-specific ribosome engagement in early development and led to the discovery of stage-specific differential engagement of zygotic RNAs with ribosomes and reduced translation efficiency of transcripts exhibiting allelic-biased expression. The novel Ribo-ITP approach will enable numerous applications by providing high coverage and high resolution ribosome occupancy measurements from ultra-low input samples including single cells.
</i>
</p>

<p>
We use Ribo-ITP to investigate diverse biological questions, often in close collaboration with domain experts. Recent highlights include:
</p>

<ol>
  <li>
    Characterization of allelic variants in the ribosome biogenesis factor AIRIM in human cerebral organoids 
    (<a href="https://www.nature.com/articles/s41556-025-01708-8" target="_blank">Ni et al., <em>Nature Cell Biology</em></a>).
  </li>
  <li>
    Application of Ribo-ITP in <em>C. elegans</em> to map dynamic changes in the translation of maternal mRNAs during the first four cell cycles of embryogenesis 
    (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12291612/" target="_blank">Shukla et al., <em>Cell Reports</em></a>).
  </li>
  <li>
Application of Ribo-ITP to uncover non-canonical translation events in ultra-low-input samples (<a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668486v1" target="_blank">Ghatpande et al., <em>bioRxiv</em></a>). Using Ribo-ITP, we profiled ribosome occupancy in microdissected hippocampal tissues and single mouse embryos, revealing thousands of translated short open reading frames (“translons”). Functional assays confirmed translation of selected translons, extending the Ribo-ITP platform to define hidden layers of the translatome in rare cell populations.

  </li>
</ol>