---
layout: page
title: Single cell ribosome profiling for characterizing translation in development, cancer and immunology
html: True
image: /img/publications/Sankey.png
summary: We developed a novel microfluidic isotachophoresis method (Ribo-ITP) to measure ribosome occupancy from ultra-low input samples including single cells. Here, we characterized transcriptome-wide ribosome occupancy during early mouse development.
permalink: /lab_research-riboitp
---


<style type="text/css">
  /* body {
    background-color: #000000;
    font-color: #FFFFFF
  }
  p {
    color: #D0E0E8
  }
  h1 {
    color: #9BC3D5
  } */
</style>

<center>
  <!-- <h1> Single cell quantification of ribosome occupancy in early mouse development </h1> -->
</center>

<center>
<iframe width="560" height="315" src="https://www.youtube.com/embed/SSHRsWqrsdc?loop=1&autoplay=1" frameborder="0" allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</center>


<p>
We are thrilled to share a new ribosome profiling approach that leverages the principles of microfluidics and isotachophoresis: <a href="https://www.nature.com/articles/s41586-023-06228-9">Ribo-ITP</a>. Our ongoing work is using Ribo-ITP to characterize translation control in biological contexts that remained beyond reach due to limited availability of biological material. Our studies encompass a broad range of areas, often involving collaborations with domain experts. This work is the culmination of six years of development by many talented scientists. See our Twitter feed for some <a href="https://twitter.com/CenikLab/status/1469077213295398919">highlights</a>. 
</p>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr"><a href="https://t.co/Y9VDhDHfa5">https://t.co/Y9VDhDHfa5</a> We developed a novel microfluidic isotachophoresis method (Ribo-ITP) to measure ribosome occupancy from ultra-low input samples including single cells. Here, we characterized transcriptome-wide ribosome occupancy during early mouse development. (1/6) <a href="https://t.co/qwF8xXJYVk">pic.twitter.com/qwF8xXJYVk</a></p>&mdash; Cenik Lab (@CenikLab) <a href="https://twitter.com/CenikLab/status/1469077213295398919?ref_src=twsrc%5Etfw">December 9, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<p>
<i> 
Translation regulation is critical for early mammalian embryonic development1. However, previous studies had been restricted to bulk measurements2 precluding precise determination of translation regulation including allele-specific analyses. To address this challenge, we developed a novel microfluidic isotachophoresis approach, named RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP), and characterized translation in single oocytes and embryos during early mouse development. We identified differential translation efficiency as a key mechanism regulating genes involved in centrosome organization and N6-methyladenosine modification of RNAs. Our high coverage measurements enabled the first analysis of allele-specific ribosome engagement in early development and led to the discovery of stage-specific differential engagement of zygotic RNAs with ribosomes and reduced translation efficiency of transcripts exhibiting allelic-biased expression. By integrating our measurements with proteomics data, we discovered that ribosome occupancy in germinal vesicle stage oocytes is the predominant determinant of protein abundance in the zygote. The novel Ribo-ITP approach will enable numerous applications by providing high coverage and high resolution ribosome occupancy measurements from ultra-low input samples including single cells.
</p>
