---
layout: page
title: Single cell ribosome profiling for characterizing translation in development, cancer and immunology
html: True
image: /img/publications/Sankey.png
summary: We developed a novel microfluidic isotachophoresis method (Ribo-ITP) to measure ribosome occupancy from ultra-low input samples including single cells.
permalink: /lab_research-riboitp
---

<center>
  <!-- <h1> Single cell quantification of ribosome occupancy in early mouse development </h1> -->
</center>

<center>
<iframe width="560" height="315" src="https://www.youtube.com/embed/SSHRsWqrsdc?loop=1&autoplay=1" frameborder="0" allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</center>


<p>
We are thrilled to share a new ribosome profiling approach that leverages the principles of microfluidics and isotachophoresis: <a href="https://www.nature.com/articles/s41586-023-06228-9">Ribo-ITP</a> (Ozadam et al., Nature):
</p>

<p>
<i> 
Translation regulation is critical for early mammalian embryonic development. However, previous studies had been restricted to bulk measurements precluding precise determination of translation regulation including allele-specific analyses. To address this challenge, we developed a novel microfluidic isotachophoresis approach, named RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP), and characterized translation in single oocytes and embryos during early mouse development. We identified differential translation efficiency as a key mechanism regulating genes involved in centrosome organization and N6-methyladenosine modification of RNAs. Our high coverage measurements enabled the first analysis of allele-specific ribosome engagement in early development and led to the discovery of stage-specific differential engagement of zygotic RNAs with ribosomes and reduced translation efficiency of transcripts exhibiting allelic-biased expression. By integrating our measurements with proteomics data, we discovered that ribosome occupancy in germinal vesicle stage oocytes is the predominant determinant of protein abundance in the zygote. The novel Ribo-ITP approach will enable numerous applications by providing high coverage and high resolution ribosome occupancy measurements from ultra-low input samples including single cells.
</i>
</p>

<p>
Our lab uses ribosome profiling and Ribo-ITP to study diverse biological questions, often in close collaboration with experts in specific fields. In partnership with Dr. Sarinay-Cenik (UT Austin), we identified naturally occurring human mutations that affect ribosome expression. Collaborating with Dr. Palazzo (University of Toronto), we investigated how the metabolic enzyme PKM influences translation by interacting with ribosomes (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10325899/">Kejiou et al., NAR</a>). In work with Dr. Buszczak (UT Southwestern), we used Ribo-ITP to characterize novel allelic variants in the ribosome biogenesis factor AIRIM in human cerebral organoids (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10802443/">Ni et al., Nature Cell Biology, in press</a>).

We have also extended Ribo-ITP to other model organisms. In <i>C. elegans</i>, we mapped dynamic changes in translation of maternal mRNAs during the first four cell cycles of embryogenesis (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11741243/">Shukla et al., bioRxiv</a>). In yeast, using selective ribosome profiling, we found that Reh1-bound ribosomes accumulate near start codons. Our data suggest that Reh1 is displaced by the elongating nascent peptide and is the final assembly factor released from the 60S ribosomal subunit during its first round of translation (<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11791190/">Musalgaonkar et al., Nature Communications</a>).
</p>