---
layout: post
title: Ultra-low input ribosome profiling for characterizing translation in development, cancer and immunology
html: True
summary: A fundamental goal of molecular biology is to characterize the dynamics of gene expression from individual cells. This knowledge will ultimately reveal the diversity of cell types that make up healthy tissues as well as changes that lead to disease states. Furthermore, mechanistic and quantitative understanding of cellular decisions will require the ability to measure analytes in individual cells with high precision.
---

<style type="text/css">
  body {
    background-color: #000000;
    font-color: #FFFFFF
  }
  p {
    color: #D0E0E8
  }
  h1 {
    color: #9BC3D5
  }
</style>


<center>
<iframe width="560" height="315" src="https://www.youtube.com/embed/SSHRsWqrsdc?loop=1&autoplay=1" frameborder="0" allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</center>

<p>
A fundamental goal of molecular biology is to characterize the dynamics of gene expression
from individual cells. This knowledge will ultimately reveal the diversity of cell types
that make up healthy tissues as well as changes that lead to disease states.
Furthermore, mechanistic and quantitative understanding of cellular decisions
will require the ability to measure analytes in individual cells with high precision.
</p>

<p>
Towards this goal, recent years have seen an explosion of methods that can measure
RNA expression and many DNA-level attributes from single cells.
However, we currently lack methods for measuring protein synthesis rates in single cells.
mRNA translation can be measured transcriptome-wide by sequencing of mRNA fragments
protected by ribosomes from RNase digestion. Despite ongoing efforts over the last ten years,
this approach (ribosome profiling) still requires approximately one million cells.
Reducing the sample input requirements for ribosome profiling remains a formidable
challenge that must be overcome to make progress on understanding the role of translation.
</p>

<p>
To address this limitation, we pioneered an on-chip isotachophoresis (ITP) method
that enables simultaneous isolation and size selection of RNA fragments protected
by ribosomes from complex lysates. This has been a wonderful collaboration with
engineers and colleagues from the National Institute of Standards and Technology
over the last 5 years. We coupled this innovation with the latest advances
in small RNA sequencing library preparation and optimization of RNase digestion conditions.
We established the feasibility of generating high quality ribosome profiling data
from as few as 100 cells. Our laboratory will further develop this approach to yield
a groundbreaking technology that will reveal the translation landscape of individual cells
from a wide range of biologically important samples.
</p>
